unit_tests/README.md

```diff
@@ -0,0 +1,63 @@
+# DTS scripts unit tests
+
+This directory contains stub scripts for testing DTS update and deploy logic
+in complex scenarios.
+
+There are some unsolved dependencies like dasharo_ectool, cbfstool, futility.
```

Comment:
I am not sure what it means. It means these should be installed on the host system where I am running tests? If this is the case, can we convert it to some `Prerequisites` section, or similar?

Reply:
I have stubbed what I had to to run these. I did not play with some fancy environment to run these test on. Executed them on host like a noob. I though it may be wiser to push it for review rather than spend 1 or 2 days more on polishing it alone.

Reply:
> I though it may be wiser to push it for review

Yes, that is great idea indeed.

The review happens right now, I cannot run the scripts, and have some questions as above, so how about we focus on answering these?

Do I need these tools to be installed in the OS? If yes, please just clearly state that which tools must be present in the environment for the tests to work.

Reply:
Added here: https://github.com/Dasharo/meta-dts/pull/95/commits/74a271545271c8e7392802e8fbfc31c8e264ac9c

Reply:
@macpijan ?

Reply:
I verified that tests can be run. Regarding environment @miczyg1 started issue on dasharo-issues https://github.com/Dasharo/dasharo-issues/issues/718

I will close this thread and merge PR into develop to prepare image for testing with DES credentials